Cargo.toml

```diff
@@ -14,4 +14,6 @@ description = "A framework for making client-side single-page apps"
 [dependencies]
 serde = "1"
 serde_json = "1"
-stdweb = "0.3"
+#stdweb = "0.3"
+
+stdweb = { path = "../stdweb" }
```

Comment:
FYI, in your `.cargo/config` in your home directory you can add this:

```
paths = [
    "/path/with/your/local/checkout/stdweb"
]
```

And then cargo will use your local copy instead of the upstream one without you having to modify the `Cargo.toml`.

Reply:
I was a little hasty about this change ) Great advice! And really much easier! 👍